### ISSUE TITLE
[iOS] How to get texture pointer

### ISSUE THREAD

**Issue by zhuyifan2013** [Bilibili/ijkplayer#2665]
I need the native texture pointer to re-create video data in unity 
But I can not get this pointer after I look through the IJKSDLGLView files.

**Comment by xinzhengzhang** [comment]
`IJK_GLES2_Renderer_renderOverlay`

**Comment by zhuyifan2013** [comment]
@xinzhengzhang  Yes, I have found this function , but when I print the texture pointer id (GLuint) , it always is 1

**Comment by zhuyifan2013** [comment]
@xinzhengzhang  

`    for (int i = 0; i < 1; ++i) {
        int plane = planes[i];

        glBindTexture(GL_TEXTURE_2D, renderer->plane_textures[i]);

        glTexImage2D(GL_TEXTURE_2D,
                     0,
                     GL_RGB,
                     widths[plane],
                     heights[plane],
                     0,
                     GL_RGB,
                     GL_UNSIGNED_SHORT_5_6_5,
                     pixels[plane]);
    }`

I think this variable is what I need ( texture pointer) : "renderer->plane_textures[i]"

**Comment by Linnkenn** [comment]
Maybe this will help you!  http://apptor.me/blog/index.html?articles/U3D/2016-12-30-%E7%BC%96%E5%86%99iOS%E7%9A%84ijkplayer%E6%8F%92%E4%BB%B6.md
